quartz-core/src/main/java/org/quartz/core/QuartzSchedulerThread.java

```diff
@@ -424,6 +440,23 @@ public void run() {
         qsRsrcs = null;
     }
 
+    private static long computeDelayForRepeatedErrors(JobStore jobStore, long acquiresFailed) {
+        long interval;
+
+        if (jobStore instanceof JobStoreSupport) {
+            interval = ((JobStoreSupport)jobStore).getDbRetryInterval();
+        } else {
+            interval = 250 * acquiresFailed;
+        }
+
+        if (interval > 5000)
```

Comment:
If the dbRetryInterval is 15 seconds, this will reduce the delay to 5 seconds.  I wonder if we'd prefer to keep the 15 seconds?   I understand that in the "else" case above, where 250 is multiplied by aquiresFailed that the interval could get huge, and therefore need to be constrained to a maximum as is being done here, but maybe that should only apply in the "else" case above.